asn1/LedgerLedgerLotocol.asn

```diff
@@ -124,8 +124,13 @@ CallSet CALL ::= {
 LedgerLedgerLotocolPacket ::= SEQUENCE {
     -- One byte type ID
     type CALL.&typeId ({CallSet}),
-    -- Allow multiple accounts to be multiplexed on one connection
-    accountId UInt32,
+    -- If the to and/or from addresses can be inferred from the
+    -- context (for example, if the connection is a bilateral
+    -- trustline), these values MAY be set to 0-byte strings
+    -- Peers MAY also use shorter aliases to avoid repeatedly
+    -- sending the full ILP Addresses over the wire
+    to Address,
+    from Address,
```

Comment:
If we switch it to strings that only contain characters that can go in addresses, the only potential difference between the `Address` format and a new one we'd come up with for this local address segment is the length. I think it would be fine to use the same schema but define in the protocol spec that the values here SHOULD be local address fragments.